database/redis/metric.go

```diff
@@ -235,6 +235,16 @@ func (connector *DbConnector) RemovePatternWithMetrics(pattern string) error {
 	return nil
 }
 
+// RemoveMetricRetention remove metric retention
```

Comment:
```suggestion
// RemoveMetricRetention remove metric retention.
```

Reply:
> 

As I understand we use "Remove" for common remove from database and we use "Cleanup" for deleting outdated/abondoned things, for example cleanup of metric retention